salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
Is this change the result of Consul changing their API between versions? If so, we may need to detect the Consul version and react appropriately here.